schema/dsrc/dsrc_schema_2-0-0.json

```diff
@@ -0,0 +1,400 @@
+        "divergePoint",
+        "downstreamStopLine",
+        "downstreamStartNode",
+        "closedToTraffic",
+        "safeIsland",
+        "curbPresentAtStepOff",
+        "hydrantPresent"
+      ]
+    },
+    "speed_limit_list": {
+      "type": "array",
+      "description": "List of SpeedLimit entries.",
+      "minItems": 1,
+      "maxItems": 9,
+      "items": {
+        "type": "object",
+        "required": [
+          "limit_type",
+          "speed"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "unknown",
+              "maxSpeedInSchoolZone",
+              "maxSpeedInSchoolZoneWhenChildrenArePresent",
+              "maxSpeedInConstructionZone",
+              "vehicleMinSpeed",
+              "vehicleMaxSpeed",
+              "vehicleNightMaxSpeed",
+              "truckMinSpeed",
+              "truckMaxSpeed",
+              "truckNightMaxSpeed",
+              "vehiclesWithTrailersMinSpeed",
+              "vehiclesWithTrailersMaxSpeed",
+              "vehiclesWithTrailersNightMaxSpeed"
+            ],
+            "description": "Type of speed limit to which a given speed refers."
+          },
+          "speed": {
+            "$ref": "#/definitions/velocity"
+          }
+        }
+      }
+    },
+    "roadway_crown_angle": {
+      "type": "integer",
+      "description": "Unit: 0.3 degrees of angle over a range of -38.1 to + 38.1 degrees. Relates the gross tangential angle of the roadway surface with respect to the local horizontal axis and is measured at the indicated part of the lane. This measurement is typically made at the crown (centerline) or at an edge of the lane path. Its typical use is to relate data used in speed warning and traction calculations for the lane segment or roadway segment in which the measurement is taken. The value -128 shall be used for unknown The value zero shall be used for angles which are between -0.15 and +0.15.",
+      "minimum": -128,
+      "maximum": 127
+    },
+    "restriction_class_id": {
+      "type": "integer",
+      "description": "Intersection-unique value to convey data about classes of users. The mapping used varies with each intersection and is defined in the MAP message if needed. The defined mappings found there are used to determine when a given class is meant. The typical use of this element is to map additional movement restrictions or rights (in both the MAP and SPAT messages) to special classes of users (trucks, high sided vehicles, special vehicles etc.). There is the general presumption that in the absence of this data, any allowed movement extends to all users.",
+      "minimum": 0,
+      "maximum": 255
+    },
+    "lane_width": {
+      "type": "integer",
+      "description": "Conveys the width of a lane in LSB units of 1 cm. Maximum value for a lane is 327.67 meters in width.",
+      "minimum": 0,
+      "maximum": 32767
+    },
+    "ia5_string": {
```

Comment:
I could not find a definition for IA5String in the DSRC... :-/

Reply:
I could not find a definition for IA5String in the DSRC... :-/

Reply:
As discussed IRL, drop the type and directly use a plain JSON string.

Reply:
Dropped and replaced by a plain string.